.github/workflows/fbgemm_gpu_cuda_release.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [ "3.8", "3.9", "3.10", "3.11" ]
-        cuda-version: [ "11.7.1", "11.8.0" ]
+        cuda-version: [ "11.7.1", "11.8.0", "12.1.1"]
```

Comment:
It looks like the CUDA 12.1 variant of PyTorch is packaged for only nightlies, not test/release.  You will have to remove this